pi-geofence/src/main/java/com/ibm/pi/geofence/GeofenceList.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2015-2016 IBM Corporation. All rights reserved.
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ibm.pi.geofence;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Encapsulates a list of geofences a returned by the PI server upon a get or sync request.
+ */
+class GeofenceList {
+    private final List<PersistentGeofence> geofences;
```

Comment:
Its Android Best Practice to prefix all member variables with an `m`

So this would be mGeofences. This helps with quickly understanding scope.